assembl/fabfile.py

```diff
@@ -876,6 +934,59 @@ def is_db_updated():
     return current in history
 
 
+def _sync_uploads_folder(local=False):
+    # Sync uploads folder
+    install_awscli()
+    with shell_env(
+        AWS_ACCESS_KEY_ID=env.aws_access_key_id,
+        AWS_SECRET_ACCESS_KEY=env.aws_secret_access_key
+    ):
+        cmd = '{}/var/uploads s3://{}/uploads'.format(env.projectpath, env.aws_bucket_name)
+        if local:
+            cmd = 's3://{}/uploads {} --delete'.format(env.aws_bucket_name, get_upload_dir())
+            venvcmd('aws s3 sync %s' % cmd)
+        else:
+            if exists(get_upload_dir()):
+                venvcmd('aws s3 cp %s' % cmd)
+            else:
+                print(red("There is no uploads folder to sync with Amazon S3!"))
+
+
+@task
+def sync_with_remote_upload_folder():
+    """
+    Syncronize the uploads folder on the host with S3 bucket
+    """
+    _sync_uploads_folder()
+
+
+@task
+def sync_with_local_upload_folder():
+    """
+    Syncronize a remote uploads folder, on S3 bucket, with the local uploads folder
+    """
+    _sync_uploads_folder(local=True)
+
+
+@task
+def reset_upload_folder():
+    if env.wsginame == 'staging.wsginame':
```

Comment:
I'm pretty sure it has to be "staging.wsgi". We have "staging.wsgi" in other places.